libraries/AP_Scripting/lua_scripts.cpp

```diff
@@ -327,7 +363,7 @@ void lua_scripts::run(void) {
 
         } else {
             gcs().send_text(MAV_SEVERITY_DEBUG, "Lua: No scripts to run");
-            hal.scheduler->delay(100);
+            hal.scheduler->delay(10000);
```

Comment:
this seems like a huge delay. if I upload a script I would expect it to start running right away. A 10s delay would cause me to upload again after a few seconds of being impatient. Can we change this to 1000 (1s)?

Reply:
We don't automatically scan that directory all the time, this message only prints once all the scripts quit. This is just a debug assistance, which less printing is better for.

Reply:
ahh yes, that makes sense. I was misinterpreting how this is used.